xml/Microsoft.VisualBasic.FileIO/FileSystem.xml

```diff
@@ -3102,6 +3102,7 @@
   
  ]]></format>
         </remarks>
+		<exception cref="T:System.NullReferenceException"></exception>
```

Comment:
- This is the wrong exception type; the method throws an ArgumentNullException.
- Each exception has to include at least one exception condition. In this case, the exception element should be: \<exception cref="T:System.ArgumentNullException">\<paramref name="file" /> is \<see langword="Nothing" /> or an empty string.\<\exception>